contracts/standard/arbitration/MultipleArbitrableTransaction.sol

```diff
@@ -0,0 +1,320 @@
+     *  @param _evidence A link to evidence or if it is short the evidence itself. Can be web link ("http://X"), IPFS ("ipfs:/X") or another storing service (using the URI, see https://en.wikipedia.org/wiki/Uniform_Resource_Identifier ). One usecase of short evidence is to include the hash of the plain English contract.
+     */
+    event Evidence(uint indexed _transactionId, Arbitrator indexed _arbitrator, uint indexed _disputeID, address _party, string _evidence);
+    
+    /** @dev To be emmited at contract creation. Contains the hash of the plain text contract. This will allow any party to show what was the original contract.
+     *  This event is used as cheap way of storing it.
+     *  @param _transactionId The index of the transaction.
+     *  @param _contractHash Keccak256 hash of the plain text contract.
+     */
+    event ContractHash(uint indexed _transactionId, bytes32 _contractHash);
+    
+    /** @dev Indicate that a party has to pay a fee or would otherwise be considered as loosing.
+     *  @param _transactionId The index of the transaction.
+     *  @param _party The party who has to pay.
+     */
+    event HasToPayFee(uint indexed _transactionId, Party _party);
+    
+    /** @dev Give a ruling for a dispute. Must be called by the arbitrator.
+     *  The purpose of this function is to ensure that the address calling it has the right to rule on the contract.
+     *  @param _disputeID ID of the dispute in the Arbitrator contract.
+     *  @param _ruling Ruling given by the arbitrator. Note that 0 is reserved for "Not able/wanting to make a decision".
+     */
+    function rule(uint _disputeID, uint _ruling) public {
+        uint transactionId = disputeTxMap[keccak256(msg.sender,_disputeID)];
+        Transaction storage transaction = transactions[transactionId];
+        require(msg.sender==address(transaction.arbitrator));
+
+        Ruling(transactionId, Arbitrator(msg.sender),_disputeID,_ruling);
+        
+        executeRuling(_disputeID,_ruling);
+    }
+
+    /** @dev Pay the arbitration fee to raise a dispute. To be called by the seller. UNTRUSTED.
+     *  Note that the arbitrator can have createDispute throw, which will make this function throw and therefore lead to a party being timed-out.
+     *  This is not a vulnerability as the arbitrator can rule in favor of one party anyway.
+     *  @param _transactionId The index of the transaction.
+     */
+    function payArbitrationFeeBySeller(uint _transactionId) payable {
+        Transaction storage transaction = transactions[_transactionId];
+        require(msg.sender == transaction.seller);
+
+
+        uint arbitrationCost = transaction.arbitrator.arbitrationCost(transaction.arbitratorExtraData);
+        transaction.sellerFee += msg.value;
+        require(transaction.sellerFee == arbitrationCost); // Require that the total pay at least the arbitration cost.
```

Comment:
It should be `>=` so it's at least. Otherwise it would block when the `arbitrator` lower the fees. Note that it was a bug in the `ArbitrableTransaction` which is no fixed.